www/content/workstation/chef_vault.md

```diff
@@ -735,10 +735,10 @@ This functionality is also available from the command line as
 
 ### Stand Alone Usage
```

Comment:
We need to confirm that this is still a use of Chef Vault that we recommend.

Reply:
Tim confirmed that it's still a thing.